src/docs/reference/core/Apis.GraphQL.Abstractions/README.md

```diff
@@ -176,6 +183,140 @@ Shown in the example above, we have an autoroutePart argument, this is registere
 
 Done.
 
+### Using default Content Type query filters
+
+In the previous section, we demonstrated how to create filters for complex requirements, allowing you to create custom filtration methods. However, in case you just need to add a simple filter on Content Type queries, there is also a simpler solution.
+
+Use this approach if you:
+
+* want to add a new filter on Content Type queries,
+* will have a database index with data for your filters,
+* you can use simple comparison (equals, contains, in...) against index values. For example, `AutoroutePartIndex.Path = filterValue`.
+
+We will cover:
+
+1. Implementing a `WhereInputObjectGraphType`.
+2. Implementing `IIndexAliasProvider`.
+3. Registering it in the `Startup` class.
```

Comment:
```suggestion
### Using default Content-Type query filters

In the previous section, we demonstrated how to create filters for complex requirements, allowing you to create custom filtration methods. However, in case you need to add a simple filter on Content-Type queries, there is also a simpler solution.

Use this approach if you:

* want to add a new filter on Content-Type queries,
* will have a database index with data for your filters,
* you can use simple comparison (equals, contains, in...) against index values. For example, `AutoroutePartIndex.Path = filterValue`.

We will cover:

1. Implementing a `WhereInputObjectGraphType`.
2. Implementing `IIndexAliasProvider`.
3. Registering it in the `Startup` class.
```